src/App.js

```diff
@@ -1,5 +1,99 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import { validateCarNameList, validateTryCount } from "./validation.js";
+import {
+  CAR_NAME_INPUT_MESSAGE,
+  TRY_COUNT_INPUT_MESSAGE,
+} from "./constants/inputMessage.js";
+import {
+  EMPTY_MESSAGE,
+  EXECUTE_RESULT_MESSAGE,
+  WINNER_MESSAGE,
+} from "./constants/outputMessage.js";
+
 class App {
-  async run() {}
+  #tryCount;
+  #cars;
+
+  constructor() {
+    this.#tryCount = 0;
+    this.#cars = [];
+  }
+
+  async run() {
+    let carNameList = await this.readCarName();
+    validateCarNameList(carNameList);
+    this.generateCars(carNameList);
+
+    const tryCount = await this.readTryCount();
+    validateTryCount(Number(tryCount));
+    this.#tryCount = Number(tryCount);
+
+    this.printGameStart();
+    this.generateGame();
+
+    const winner = this.generateGameResult();
+    this.printGameResult(winner);
+  }
+
+  async readCarName() {
+    return await Console.readLineAsync(CAR_NAME_INPUT_MESSAGE);
+  }
+
+  async readTryCount() {
+    return await Console.readLineAsync(TRY_COUNT_INPUT_MESSAGE);
+  }
+
+  generateCars(carNameListProp) {
+    const carNameList = carNameListProp.split(",");
+    carNameList.forEach((elem) => this.#cars.push(new Car(elem)));
```

Comment:
```suggestion
    carNameList.forEach((carName) => this.#cars.push(new Car(carName)));
```
사소하지만 `elem`보다는 `carNameList`를 순회하는 거니깐 `carName`으로 조금 더 명확하게 나타내도 좋을 것 같아요!